third_party/move/move-compiler-v2/transactional-tests/tests/misc/tuple_swap.exp

```diff
@@ -0,0 +1,36 @@
+processed 12 tasks
+
+task 1 'run'. lines 108-108:
+return values: 2, 1
+
+task 2 'run'. lines 110-110:
+return values: 30, 40, 20, 10
```

Comment:
For future reference, it would be better to add `asserts` in the .move file so that inconsistency is locally checkable.  It's too easy to have a slight change in test output go unnoticed among other changes.

Reply:
Fair point.